### DISCUSSION TITLE
How do I specify an aspect ratio when running smplayer from the command line?

### DISCUSSION THREAD

**Discussion by supplementfacts2** [smplayer-dev/smplayer#1007] [Q&A]
I know an mplayer command that specifies an aspect ratio looks like this:

`mplayer -aspect 4:3 "c:\folder\file.mp4"`

But, despite a lot of googling, I can't find any way to make the smplayer command specify an aspect ratio. Here are the ones I've tried:

`smplayer -aspect 4:3 "c:\folder\file.mp4"`
`smplayer -aspect 4/3 "c:\folder\file.mp4"`
`smplayer -aspect=4:3 "c:\folder\file.mp4"`
`smplayer -aspect=4/3 "c:\folder\file.mp4"`
`smplayer -aspect:4:3 "c:\folder\file.mp4"`
`smplayer -aspect:4/3 "c:\folder\file.mp4"`
`smplayer --aspect 4:3 "c:\folder\file.mp4"`
`smplayer --aspect 4/3 "c:\folder\file.mp4"`
`smplayer --aspect=4:3 "c:\folder\file.mp4"`
`smplayer --aspect=4/3 "c:\folder\file.mp4"`
`smplayer --aspect:4:3 "c:\folder\file.mp4"`
`smplayer --aspect:4/3 "c:\folder\file.mp4"`
`smplayer aspect 4:3 "c:\folder\file.mp4"`
`smplayer aspect 4/3 "c:\folder\file.mp4"`
`smplayer aspect=4:3 "c:\folder\file.mp4"`
`smplayer aspect=4/3 "c:\folder\file.mp4"`
`smplayer aspect:4:3 "c:\folder\file.mp4"`
`smplayer aspect:4/3 "c:\folder\file.mp4"`
`smplayer "c:\folder\file.mp4" -aspect 4:3`
`smplayer "c:\folder\file.mp4" -aspect 4/3`
`smplayer "c:\folder\file.mp4" -aspect=4:3`
`smplayer "c:\folder\file.mp4" -aspect=4/3`
`smplayer "c:\folder\file.mp4" -aspect:4:3`
`smplayer "c:\folder\file.mp4" -aspect:4/3`
`smplayer "c:\folder\file.mp4" --aspect 4:3`
`smplayer "c:\folder\file.mp4" --aspect 4/3`
`smplayer "c:\folder\file.mp4" --aspect=4:3`
`smplayer "c:\folder\file.mp4" --aspect=4/3`
`smplayer "c:\folder\file.mp4" --aspect:4:3`
`smplayer "c:\folder\file.mp4" --aspect:4/3`
`smplayer "c:\folder\file.mp4" aspect 4:3`
`smplayer "c:\folder\file.mp4" aspect 4/3`
`smplayer "c:\folder\file.mp4" aspect=4:3`
`smplayer "c:\folder\file.mp4" aspect=4/3`
`smplayer "c:\folder\file.mp4" aspect:4:3`
`smplayer "c:\folder\file.mp4" aspect:4/3`

None of them play the video in the specified aspect ratio, although some of them do at least play the video. Does anyone know how to do it? Thanks!

**Comment by smplayer-dev** [comment]
`smplayer -actions aspect_4:3 video.mp4`

**Comment by supplementfacts2** [comment]
Thank you, but there is still a problem. I used 4:3 as an example, but the command doesn't seem to work for other aspect ratios: e.g., 2:1 or (just to choose a strange one) 7:2. Is there a way to open a file with 2:1? Thanks again!

**Comment by smplayer-dev** [comment]
I'm afraid only the aspect ratios listed in the menu Video -> Aspect ratio can be used this way.
